lib/jsbi.ts

```diff
@@ -1820,27 +1822,27 @@ class JSBI extends Array {
   }
 
   // Digit helpers.
-  __digit(i) {
+  __digit(i): number {
```

Comment:
```suggestion
  __digit(i: number): number {
```

Same for the next three functions below.

Reply:
Done.